dbt_subprojects/daily_spellbook/models/xrpl/transfers/tokens_xrpl_amm_deposits.sql

```diff
@@ -0,0 +1,303 @@
+{{
+    config(
+        schema = 'tokens_xrpl_amm_deposits',
+        alias = 'transfers',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date','unique_key'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')],
+        post_hook='{{ expose_spells(\'["xrpl"]\',
+                                    "sector",
+                                    "tokens",
+                                    \'["krishhh"]\') }}'
```

Comment:
```suggestion
```

Reply:
lets remove expose spells macro from all individual transfer type models, and save it only for the final union view. this will encourage using that spell the most.